sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTest.java

```diff
@@ -364,6 +363,15 @@ public void cancelCertificateOperation(HttpClient httpClient, CertificateService
         });
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void cancelCertificateCreationOperation(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
+        createCertificateClient(httpClient, serviceVersion);
+        cancelCertificateCreationOperationRunner((certName) -> {
```

Comment:
We should rename this to `createCertificateOperation` and have the method above it be called `createCertificateOperationThroughPoller` or something like that, feels more descriptive of what both methods do.
```suggestion
    public void cancelCertificateOperation(HttpClient httpClient, CertificateServiceVersion serviceVersion) {
        createCertificateClient(httpClient, serviceVersion);
        cancelCertificateOperationRunner((certName) -> {
```

Reply:
Also, please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/16691/files#r513124806) about the implementation of this test case.